tests/js/spec/views/dashboardsV2/widgetQueries.spec.jsx

```diff
@@ -302,6 +320,114 @@ describe('Dashboards > WidgetQueries', function () {
     expect(childProps.tableResults[1].data[0].title).toBeDefined();
   });
 
+  it('can send big number result queries', async function () {
+    const tableMock = MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/eventsv2/',
+      body: {
+        meta: {'sdk.name': 'string'},
+        data: [{'sdk.name': 'python'}],
+      },
+    });
+
+    let childProps = undefined;
+    const wrapper = mountWithTheme(
+      <WidgetQueries
+        api={api}
+        widget={{
+          title: 'SDK',
+          interval: '5m',
+          displayType: 'big_number',
+          queries: [{conditions: 'event.type:error', fields: ['sdk.name'], name: 'sdk'}],
+        }}
+        organization={initialData.organization}
+        selection={selection}
+      >
+        {props => {
+          childProps = props;
+          return <div data-test-id="child" />;
+        }}
+      </WidgetQueries>,
+      initialData.routerContext
+    );
+    await tick();
+    await tick();
+
+    // Child should be rendered and 2 requests should be sent.
```

Comment:
Am I missing something or is the test below asserting that only 1 request was sent